app/policies/user_policy.rb

```diff
@@ -13,6 +13,10 @@ def edit?
     admin? || user == record
```

Comment:
Nem tudok pont arra a sorra komentet fűzni, mert béna vagyok, de
Ha megszűnt a show endpont, akkor a policy is alkalmazkodjon ehhez.

Reply:
policy-t ne töröljünk csak azért, mert éppen nincs hozzá endpont